scripts/store_user_files.sh

```diff
@@ -11,20 +11,6 @@ DIR=
 TARGETDIR=
 CP=
 
-# Parse arguments.
```

Comment:
The file `scripts/store_user_files.sh` should also be completely deleted.

Reply:
Done!